data-prepper-plugins/kafka-connect-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafkaconnect/extension/WorkerProperties.java

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.kafkaconnect.extension;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.kafka.connect.runtime.WorkerConfig;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+public class WorkerProperties {
+    private static final String KEY_CONVERTER = "org.apache.kafka.connect.json.JsonConverter";
+    private static final String KEY_CONVERTER_SCHEMAS_ENABLE = "false";
+    private static final String VALUE_CONVERTER_SCHEMAS_ENABLE = "false";
+    private static final String VALUE_CONVERTER = "org.apache.kafka.connect.json.JsonConverter";
+    private static final Integer OFFSET_STORAGE_PARTITIONS = 25;
+    private static final Long OFFSET_FLUSH_INTERVAL_MS = 60000L;
+    private static final Long OFFSET_FLUSH_TIMEOUT_MS = 5000L;
+    private static final Integer STATUS_STORAGE_PARTITIONS = 5;
+    private static final Long HEARTBEAT_INTERVAL_MS = 3000L;
+    private static final Long SESSION_TIMEOUT_MS = 30000L;
+    private static final String DEFAULT_GROUP_ID = "localGroup";
+    private static final String DEFAULT_CLIENT_ID = "localClient";
+    private static final String DEFAULT_CONFIG_STORAGE_TOPIC = "config-storage-topic";
+    private static final String DEFAULT_OFFSET_STORAGE_TOPIC = "offset-storage-topic";
+    private static final String DEFAULT_STATUS_STORAGE_TOPIC = "status-storage-topic";
+    private final Integer offsetStorageReplicationFactor = -1;
+    private final Integer configStorageReplicationFactor = -1;
+    private final Integer statusStorageReplicationFactor = -1;
+    private String keyConverter = KEY_CONVERTER;
+    private String keyConverterSchemasEnable = KEY_CONVERTER_SCHEMAS_ENABLE;
+    private String valueConverter = VALUE_CONVERTER;
+    private String valueConverterSchemasEnable = VALUE_CONVERTER_SCHEMAS_ENABLE;
+
+    @JsonProperty("group_id")
+    private String groupId = DEFAULT_GROUP_ID;
+    @JsonProperty("config_storage_topic")
+    private String configStorageTopic = DEFAULT_CONFIG_STORAGE_TOPIC;
+    @JsonProperty("offset_storage_topic")
+    private String offsetStorageTopic = DEFAULT_OFFSET_STORAGE_TOPIC;
+    @JsonProperty("status_storage_topic")
+    private String statusStorageTopic = DEFAULT_STATUS_STORAGE_TOPIC;
+    @JsonProperty("client_id")
+    private String clientId = DEFAULT_CLIENT_ID;
+    @JsonProperty("offset_storage_partitions")
+    private Integer offsetStoragePartitions = OFFSET_STORAGE_PARTITIONS;
+    @JsonProperty("offset_flush_interval_ms")
```

Comment:
See my other comment about using `Duration` instead. This will help yield consistent configurations.

Reply:
Sure